components/timeline/assets/Timeline.scss

```diff
@@ -0,0 +1,120 @@
+.hods-timeline {
+  /* Write base styles for Timeline here*/
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+
+  & p {
+    text-decoration: inherit;
+    font-size: inherit;
+  }
+
+  &__item {
+    padding-bottom: 10px;
+    padding-left: 20px;
+    position: relative;
+
+    &:hover {
+      background-color: #E3E3E3;
+    }
+
+    &:before {
+      background-color: #1d70b8;
+      content: "";
+      height: 5px;
+      left: 0;
+      position: absolute;
+      top: 10px;
+      width: 15px;
+      z-index: 1;
+    }
+  }
+
+  &:before {
+    background-color: #1d70b8;
+    content: "";
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 10px;
+    width: 5px;
+    z-index: 1;
+  }
+
+  &__title {
+    font-family: "GDS Transport", Arial, sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-weight: 700;
+    font-size: 16px;
+    font-size: 1rem;
+    line-height: 1.25;
+    display: inline;
+  }
+
+  &__by {
+    font-family: "GDS Transport", Arial, sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-weight: 400;
+    font-size: 16px;
+    font-size: 1rem;
+    line-height: 1.25;
+    color: #6f777b;
+    display: inline;
+    margin: 0;
+  }
+
+  &__date {
+    font-family: "GDS Transport", Arial, sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-weight: 400;
+    font-size: 14px;
+    font-size: 0.875rem;
+    line-height: 1.14286;
+    margin-top: 5px;
```

Comment:
We probably want a responsive mixin here: https://github.com/UKHomeOffice/hods-poc/blob/master/components/status-banner/assets/StatusBanner.scss#L34